packages/icons/scripts/web.ts

```diff
@@ -114,9 +115,9 @@ function buildDeclarationFile(components: Component[]): string {
 
     export interface IconProps extends React.SVGProps<SVGSVGElement> {
       /**
-       * Choose between the 16 or 24 pixels size. Defaults to \`24\`.
+       * Choose between the 16, 24 or 32 pixels size. Defaults to \`24\`.
        */
-      size?: '16' | '24';
+      size?: '16' | '24' | '32';
```

Comment:
Nice, that looked simple enough 😅 did you check if TS was still intelligent enough to only show valid sizes for icons?

For example if `PaidOut` is in 16 and 24 it should show these, but not 32. But `AmericanExpress` should show only 24 (payment method) and 32 (card scheme)

Reply:
Good catch! I updated the `web.ts` to return a generic `IconProps` interface with only the available sizes.